City-of-Helsinki/helsinki-paatokset
# PP-370: Add next meeting livestream tag

**To test**
- Checkout branch, run `make drush-cr`
- Run `make ahjo-migrations` (if you don't have policymakers and meetings) and `drush migrate-import paatokset_meeting_videos --update`
- Go to https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto
  - There should be a label that displays when the next meeting will be above the most recent video 
- Read the code for the logic and check that it makes sense. The label should display up until 24:00 of the day before the livestream starts.
